xbridge-confs/trittium--v4.0.2.1.conf

```diff
@@ -0,0 +1,21 @@
+[TRTT]
+Title=Trittium
+Address=
+Ip=127.0.0.1
+Port=31002
+Username=
+Password=
+AddressPrefix=65
+ScriptPrefix=19
+SecretPrefix=239
+COIN=100000000
+MinimumAmount=0
+TxVersion=1
+DustAmount=0
+CreateTxMethod=BTC
+GetNewKeySupported=true
+ImportWithNoScanSupported=true
+MinTxFee=0
+BlockTime=60
+FeePerByte=25
+Confirmations=0
```

Comment:
I'm not sure why this `xbridge-confs/trittium--v4.0.2.1.conf` file is being added. `manifest-latest.json` doesn't ever reference this config file. 😕